apiary.apib

```diff
@@ -175,22 +221,83 @@ Note: Rewards generated in the Sandbox environment will be delivered to the API
 
     + Body
       {
-          "order": {
-            "id": "QABSTARTSFSIO",
-            "external_id": "[ID FROM CLIENT]",
-            "payment": {
-              "subtotal": order.amount_dollars,
-              "total": order.price_dollars,
-              "fees": order.fee_dollars
-            },
-            "rewards": [{
-              id: "DDABSUKSFSTY"
-            }]
-          }
+        "order": {
+          "id": "QABSTARTSFSIO",
+          "external_id": "[ID FROM CLIENT]",
+          "status": "EXECUTED",
+          "payment": {
+            "subtotal": order.amount_dollars,
+            "total": order.price_dollars,
+            "fees": order.fee_dollars
```

Comment:
I think since this is a sample response we should fill this with some value (so they know the expected format). Someone actually mentioned that they couldn't parse this JSON when using the mock server because of these fields

Reply:
yeah, good call